xbmc/settings/AdvancedSettings.h

```diff
@@ -288,6 +288,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     bool m_curlDisableIPV6;
     bool m_curlDisableHTTP2;
 
+    std::string m_catrustfile;
```

Comment:
Can you use camelCase? My cat rust file won't stop chasing corrosive data.

Reply:
done

Reply:
I would also change `cacert`. Um... my cace is realtime?